pkg/cloud-init/cloud-init_test.go

```diff
@@ -502,6 +502,7 @@ var _ = Describe("CloudInit", func() {
 			})
 		})
 	})
+
```

Comment:
I think I saw a unit test here before. Was it removed?

Reply:
Yes, it doesn't apply anymore since the validity check moved elsewhere...
I'm still trying to figure out which function could/should be unit-tested.

Reply:
Added to `manager_test.go`, pushed and rebased

Reply:
I just wanted to see that the file that is being created here has an expected size on the disk.

Reply:
Fix force-pushed!